src/qt/sendcoinsrecipient.h

```diff
@@ -33,8 +57,8 @@ class SendCoinsRecipient
     CAmount amount;
     // If from a payment request, this is used for storing the memo
     QString message;
-    // Keep the payment request around as a serialized string to ensure
-    // load/store is lossless.
+    // If building with BIP70 is disabled, keep the payment request around as
```

Comment:
This is no longer an "if" :smiley:

Reply:
Oops, accident revert added in a rebase. Fixed.